### PR TITLE
adds our old ion laws and ups ion law weight from 15 to 20

### PR DESCRIPTION
**Pull request by Amelia0010**
:cl:  Dagdammit
rscadd: New and improved Ion-laws, makes ion-laws more meaningful
tweak: ion-law weight has been upped
/:cl:

### CODE REVIEW

**Comment by Amelia0010** [comment]
and yes, I know this isn't entirely modularized but the strings proc checks for the /strings/ folder and marking it clearly as yogs_ion_laws is easier, since other systems depend on the old file.

**Comment by Amelia0010** [comment]
I copied the json from the old PR and I have no clue why it's failing travis

File: strings/yogs_ion_laws.json [hunk 1]
```diff
@@ -0,0 +1,376 @@
...
+        "ASSISTANTS",
+        "BARTENDERS",
+        "CHEFS",
+        "CLOWNS",
+        "MIMES",
+        "CHAPLAINS",
+        "HEADS OF STAFF",
+        "CYBORGS",
+        "ARTIFICAL INTELLIGENCES",
+        "MASKED INDIVIDUALS"
```
Comment (AshCorr) [inline strings/yogs_ion_laws.json]: You're missing a comma here.

**Comment by ThatLing** [comment]
this is code

**Pull request merged**